docs/2-retrieving-data-as-objects.md

```diff
@@ -4,17 +4,17 @@
 The `FirebaseObjectObservable` is not created by itself, but through the `AngularFire.database` service. 
 The guide below demonstrates how to retrieve, save, and remove data as objects.
 
-## Injecting the AngularFire service
+## Injecting the AngularFireDatabase module
 
 **Make sure you have bootstrapped your application for AngularFire2. See the Installation guide for bootstrap setup.**
 
-AngularFire is an injectable service, which is injected through the constructor of your Angular component or `@Injectable()` service.
+AngularFireDatabase is an @NgModule, which is injected through the constructor of your Angular component or `@Injectable()` service.
```

Comment:
`AngularFireDatabase` is a service, `AngularFireDatabaseModel` is the containing module.